src/Criterion.cpp

```diff
@@ -2,26 +2,26 @@
 #include <iostream>
```

Comment:
fyi we all have these changes in our PR, it is likely to cause annoying merge conflict after the first will be merged !